server/plugin/command.go

```diff
@@ -136,23 +136,28 @@ func (p *Plugin) postCommandResponse(args *model.CommandArgs, text string) {
 	p.client.Post.SendEphemeralPost(args.UserId, post)
 }
 
-func (p *Plugin) getMutedUsernames(userInfo *GitHubUserInfo) []string {
+func (p *Plugin) getMutedUsernames(userInfo *GitHubUserInfo) ([]string, error) {
 	var mutedUsernameBytes []byte
 	err := p.store.Get(userInfo.UserID+"-muted-users", &mutedUsernameBytes)
 	if err != nil {
-		return nil
+		return nil, err
 	}
 	mutedUsernames := string(mutedUsernameBytes)
 	var mutedUsers []string
 	if len(mutedUsernames) == 0 {
-		return mutedUsers
+		return mutedUsers, nil
 	}
 	mutedUsers = strings.Split(mutedUsernames, ",")
-	return mutedUsers
+	return mutedUsers, nil
 }
 
-func (p *Plugin) handleMuteList(args *model.CommandArgs, userInfo *GitHubUserInfo) string {
-	mutedUsernames := p.getMutedUsernames(userInfo)
+func (p *Plugin) handleMuteList(_ *model.CommandArgs, userInfo *GitHubUserInfo) string {
+	mutedUsernames, err := p.getMutedUsernames(userInfo)
+	if err != nil {
+		p.client.Log.Error("error occurred getting muted user.", "Error", err)
+		return "Some error occurred getting muted users. Please try again later"
```

Comment:
Can we change these messages to "An error occurred...".  (all instances)

Reply:
@wiggin77  Done